samples/matter/light_switch/src/app_task.cpp

```diff
@@ -38,32 +41,34 @@
 #include <zephyr/logging/log.h>
 #include <zephyr/zephyr.h>
 
+LOG_MODULE_DECLARE(app, CONFIG_MATTER_LOG_LEVEL);
+
 using namespace ::chip;
 using namespace ::chip::app;
 using namespace ::chip::Credentials;
 using namespace ::chip::DeviceLayer;
 
-LOG_MODULE_DECLARE(app, CONFIG_MATTER_LOG_LEVEL);
-
 namespace
 {
-constexpr EndpointId kLightSwitchEndpointId = 1;
-constexpr EndpointId kLightEndpointId = 1;
 constexpr uint32_t kFactoryResetTriggerTimeout = 3000;
-constexpr uint32_t kFactoryResetCancelWindow = 3000;
+constexpr uint32_t kFactoryResetCancelWindowTimeout = 3000;
 constexpr uint32_t kDimmerTriggeredTimeout = 500;
 constexpr uint32_t kDimmerInterval = 300;
-constexpr uint32_t kIdentifyBlinkRateMs = 500;
 constexpr size_t kAppEventQueueSize = 10;
+constexpr EndpointId kLightSwitchEndpointId = 1;
+constexpr EndpointId kLightEndpointId = 1;
 
 K_MSGQ_DEFINE(sAppEventQueue, sizeof(AppEvent), kAppEventQueueSize, alignof(AppEvent));
+k_timer sFunctionTimer;
+k_timer sDimmerPressKeyTimer;
+k_timer sDimmerTimer;
 
 Identify sIdentify = { kLightEndpointId, AppTask::IdentifyStartHandler, AppTask::IdentifyStopHandler,
 		       EMBER_ZCL_IDENTIFY_IDENTIFY_TYPE_VISIBLE_LED };
 
 LEDWidget sStatusLED;
 LEDWidget sIdentifyLED;
-#if NUMBER_OF_BUTTONS == 4
+#if NUMBER_OF_LEDS == 4
 LEDWidget sUnusedLED;
```

Comment:
sUnusedLED->sFactoryResetSignalLED and sUnusedLED_1->sFactoryResetSignalLED_1?

Reply:
Implemented a wrapper for factory reset LEDs.